### ISSUE TITLE
Players breaking in newer Kodi versions

### ISSUE THREAD

**Issue by mrgsi** [a4k-openproject/json.openplayers#92]
Anyone know what is breaking some of the players in Kodi 19.4-nightlies and Kodi 20?
Any Oath forks like Homelander are broken as is the Scrubs player. The add-ons themselves work fine and not all players are affected. The Homelander player and the Crew player are coded the same way but the Crew player works fine in Kodi 20.
It's something that was added to Kodi in any versions beyond the recommended 19.4 version.

**Comment by JayDee696969** [comment]
Yep, noticed that also. Same players that worked in Matrix dont work in Nexus. Wish I was better at writing players, but stull not sure if its just because Trakt is down, or the players arent written correctly. Waiting for Trakt to come back up before I try to figure it out... https://github.com/jurialmunkey/plugin.video.themoviedb.helper/issues/957

**Comment by mrgsi** [comment]
No it's something to do with a change in Kodi just after the 19.4 recommended version. I installed a 19.4 nightly version about a month ago and noticed the issue. Whatever it is has been carried forward to the most recent Kodi 20 RC! version.

**Comment by JayDee696969** [comment]
> No it's something to do with a change in Kodi just after the 19.4 recommended version. I installed a 19.4 nightly version about a month ago and noticed the issue. Whatever it is has been carried forward to the most recent Kodi 20 RC! version.

Lol, pretty sure theres no such thing as a 19.4 nightly version.. Hmm

**Comment by mrgsi** [comment]
Well then, let me educate you. Go to Kodi.tv and into the downloads. Instead of "Recommended" choose "Development Builds". Those versions are called Nightly versions. The Kodi team pushes out nightly builds all the time. The Windows version of 19.4 has 4 for Dec on Dec 4th, 12th, 15th and 16th. They are minor bug fixes, but not enough to offer them as a new 19.5 version.

**Comment by JayDee696969** [comment]
> Well then, let me educate you. Go to Kodi.tv and into the downloads. Instead of "Recommended" choose "Development Builds". Those versions are called Nightly versions. The Kodi team pushes out nightly builds all the time. The Windows version of 19.4 has 4 for Dec on Dec 4th, 12th, 15th and 16th. They are minor bug fixes, but not enough to offer them as a new 19.5 version.

Oh, ok. I was under the assumption that once a version is final any nighties would be nightly updates for the next release version.

**Comment by mrgsi** [comment]
Fix found. The players for Exodus Fork add-ons need to be adjusted by replacing all the "`&amp;`" in the code with just "&" to work in Kodi 19.5 and Kodi 20. Life is good again, lol
